javascript-data-model/order-history.js

```diff
@@ -0,0 +1,69 @@
+var orderHistory = [
+  {
+    item: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    itemCost: 31.55,
+    orderTotal: 34,
+    orderPlaced: 'August 4, 2020',
+    delivered: 'August 8, 2020',
+    returnWindow: 'September 7, 2020',
+    shipto: 'JS Masher',
+    orderNumber: '114-394',
+    image: 'url javascript'
+  },
```

Comment:
In this case, the data for the order itself is intermixed with the data concerning the items ordered, which makes this data model ambiguous and difficult to use. You should have a property of the object for the order that contains the data for the item(s) ordered by the user.